backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt

```diff
@@ -0,0 +1,34 @@
+package app.ehrenamtskarte.backend.verification.database
+
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IntIdTable
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+
+const val HASH_LENGTH = 32 // TODO check this once the frontend implementation is done
+const val TOTP_SECRET_LENGTH = 20
+
+@ExperimentalUnsignedTypes
+object Cards : IntIdTable() {
```

Comment:
Should we have an additional id if we have a unique hash? Or could the has act as id?

Reply:
Well, there is an additional index on the hash. I thought it might be a good idea to have an additional, easier to check primary key in case we once would want to make table joins or something. But I am not committed into that.